Umbraco-Cloud/Troubleshooting/Deployments/Structure-Error/index.md

```diff
@@ -24,94 +20,124 @@ Example:
 
-![After extraction error](images/visualization2.png)
+For this example, it’s decided that the Document Type currently used on the Live environment is the one we want to use going forward.
 
-On a deployment between Umbraco Cloud environments, all the `.uda` files in the `/data/revision` folder will get synced. For this project, this means that both the Development and the Live environments will have two different `.uda` files for the Document Type – the only thing that’s different between the two files are the GUID since they were created in different environments.
+![Visual representation of the error message](images/visualizing.png) - *A visual representation of the error message*
 
-**NOTE**: This is when you will see an extraction error like the one shown at the beginning of this article. It is simply not possible to add types with the same alias in the database.
+In order to figure out which of the two colliding `.uda` files is the one for the Document Type being used on the Live environment follow these steps:
 
-### Choosing the correct UDA file
+1. Access **Kudu** on the Live environment
+2. Use the CMD console (found under the 'Debug console' menu) to navigate to your `site/wwwroot/data/revision` folder
+3. Remove the colliding `.uda` files mentioned in the error message
+4. Go back to the `/wwwroot/data` folder and run this command: `echo > deploy-export` in the console
+5. This will regenerate a `.uda` file for the Document Type being used on the Live environment
+6. Run the command: `echo > deploy` in the same folder, to make sure everything is extracting correctly
 
-The next step is to decide which of these Document Types is the correct one. For this project, it’s decided that the Document Type created in the Live environment (DocType 1 and document-type__1.uda) is the correct one.
+![Finding correct UDA file](images/visualizing-2.png)
 
-In order to figure out which of the two colliding `.uda` files are the file for the Document Type created on the Live environment follow these steps:
+You now know which `.uda` file you want. In this case: `document-type__1.uda`.
 
-1.    Access **Kudu** for the Live environment / the environment where the correct Document Type is
-2. Use the CMD console (found under the 'Debug console' menu) to navigate to your `site/wwwroot/data/` folder
-3.    Remove both colliding `.uda` files from the `/data/revision` folder in the `/wwwroot` folder.
-4.    In `/wwwroot/data` run this command: `echo > deploy-export` 
-5.    This will generate a `.uda` file for the Document Type, and this will be the correct one
-6.    Run `echo > deploy` in the same folder, to make sure everything is extracting correctly
+## Removing the unused file
 
-![Finding correct UDA file](images/visualization3.png)
+We strongly recommend that you resolve this locally since this will ensure that the changes you make are added to your Git repositories.
```

Comment:
```suggestion
:::warning
We strongly recommend that you resolve this locally since this will ensure that the changes you make are added to your Git repositories. Otherwise you may end up having the same problem next time you deploy!
:::
```